SU-HSDO/suhumsci
# HSD8-677 Full width lead font

# READY FOR REVIEW

# Summary
- Override stanford_basic lead font max-width. It made some awkward looking styles
![screenshot](https://user-images.githubusercontent.com/7185045/60927681-97918500-a25f-11e9-9d9f-f1f62e33a475.png)


# Needed By (Date)
- whenever

# Urgency
- low

# Steps to Test
1. checkout this branch
1. create a basic page with a text area paragraph
1. use the "Lead Font" style option for some text
1. ensure it goes full width.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
